Contributing-code-to-Oppia.md

```diff
@@ -1,306 +1,211 @@
-       git fetch upstream
-       git merge upstream/develop
-       ...[fix the conflicts -- see https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line]...
-       ...[make sure the tests pass before committing]...
-       git commit -a
-       git push origin new-branch-name
-       ```
+* [#8472](https://github.com/oppia/oppia/issues/8472) (Migrating AngularJS files to Angular 2+)
 
-       **Note:** Do not squash commits when you merge develop into your branch! This often happens when you are using an graphical interface instead of the command line. Squashing commits can confuse GitHub and cause the changes you merge in from develop to be treated as changes you made. This makes PRs really hard to review!
+* [#10798](https://github.com/oppia/oppia/issues/10798) (Fixing end-to-end tests, which are written with protractor)
 
-     - Make a new commit addressing the comments you agree with, and push it to the same branch. (Continue to use descriptive commit messages, or something like "Address review comments" if you're addressing many disparate review comments in the same commit.) **You do not need to close your PR and create a new one -- it's fine to push new commits to the existing PR.**
-       - **Always make commits locally, and then push to GitHub.** Don't make changes using the online GitHub editor -- this bypasses lint/presubmit checks, and will cause the code on GitHub to diverge from the code on your machine.
-       - **Never force-push changes to GitHub once reviews have started.** This will delay your review because it overwrites history on GitHub and makes the incremental changes harder to review. It may also lead to the PR being closed.
-      - As you are making changes, track them by replying to each comment via the Files Changed tab, **choosing the "Start a review" option** for the first comment. Each reply should be either "Done" or a response explaining why the corresponding suggestion wasn't implemented. Also, please **do not** mark the comment as resolved, since this just makes it harder to actually read the comment thread. When you've
-responded to all comments, submit the review to add all your messages to the main thread.
-       - **Tip:** If a reviewer asks questions about the "why" behind something, consider proactively adding a clear comment above the relevant line in your code, since the fact that the reviewer had to ask suggests that at least one developer doesn't understand what is going on from the code alone. Otherwise, you'll probably get a follow-up review comment asking you to leave a code comment anyway :)
-       - **Note:** If any comments require follow-up PRs, please file an issue to track the necessary changes, and mention the issue number in the relevant review comment thread. (You should do this automatically for any deferred changes; reviewers may defer approval until you do.)
+* [#4057](https://github.com/oppia/oppia/issues/4057) (frontend; Karma tests)
```

Comment:
I thought this was being done as a gsoc project; maybe confirm with @nithusha21 @kevintab95 whether it should still be offered.

Reply:
I think this should no longer be a starter task, but I'm checking with the GSoC mentors. If they agree, I'll remove this line

Reply:
I've removed it as a starter issue after conferring with @nithusha21